core/src/test/scala/integration/kafka/api/ConsumerBounceTest.scala

```diff
@@ -147,6 +148,230 @@ class ConsumerBounceTest extends IntegrationTestHarness with Logging {
     }
   }
 
+  /**
+   * Consumer is closed while cluster is healthy. Consumer should complete pending offset commits
+   * and leave group. New consumer instance should be able join group and start consuming from
+   * last committed offset.
+   */
+  @Test
+  def testClose() {
```

Comment:
`ConsumerBounceTest` now takes 3m10s, up from 1m4s on my laptop. Which is rather high, I have to admit. The times are (the last two are the older tests):

* testClose 	10.099s
* testCloseDuringRebalance 	14.021s
* testCloseDuringRebalanceBrokersUnavailable 	14.171s
* testCloseNoTimeoutWithClusterFailure 	12.090s
* testCloseTimeoutWithClusterFailure 	9.859s
* testCloseWithCoordinatorFailureUsingGroupManagement 	14.024s
* testCloseWithCoordinatorFailureUsingManualAssignment 	10.080s
* testCloseWithRandomBrokerFailures 	39.924s
* testConsumptionWithBrokerFailures 	42.199s
* testSeekAndCommitWithBrokerFailures 	23.883s

I will remove `testCloseWithRandomBrokerFailures` since not sure it is worth spending 40 seconds on, system tests should come across these scenarios. Will try and remove some more cases to bring it below 2 minutes.